pkgs/applications/science/astronomy/galario/default.nix

```diff
@@ -0,0 +1,75 @@
+{ stdenv
+, fetchurl
+, cmake
+, fftw
+, fftwFloat
+, enablePython ? false
+, pythonPackages
+, llvmPackages
+}:
+
+stdenv.mkDerivation rec {
+  name = "galario-${version}";
+  version = "1.2.1";
+
+  srcs = [
+    (fetchurl {
```

Comment:
You can use `fetchFromGitHub` then you don't need to build the URl yourself.

Reply:
`fetchFromGitHub` does not work for multiple sources: all source file are unpacked into the same "source/" directory, so the second file fails to unpack with a permission error.